wemake_python_styleguide/violations/consistency.py

```diff
@@ -1748,3 +1750,34 @@ class OperationSignNegationViolation(ASTViolation):
 
     error_template = 'Found wrong operation sign'
     code = 346
+
+
+@final
+class LiteralNoneViolation(ASTViolation):
```

Comment:
After creating so many violations about types - I have decided to make a special violation group and a visitor for them. Let's move this one into `violations/annotations.py` and use a new code (`WPS7xx`, I guess)